include/openspace/navigation/orbitalnavigator.h

```diff
@@ -233,6 +233,15 @@ class OrbitalNavigator : public properties::PropertyOwner {
 
     properties::BoolProperty _invertMouseButtons;
 
+    properties::BoolProperty _shouldRotateAroundUp;
```

Comment:
More a general question then just for here;  should it be called "rotate around axis" rather than up?  Since we can choose what "up" is, it might not match with the persons understanding of "up" necessarily

Reply:
I get your point. We should probably discuss it. 

The reason I chose to use "Up" is that I consider the axis property to be a "temporary" property, until each scene graph node knows about its up direction (https://github.com/OpenSpace/OpenSpace/issues/740)

However ,I guess that could be seen as a 4th axis option and that we could still want to orbit around any of the 3 axes

Reply:
Ah.. in that case I withdraw my comment.  Using the Up vector of the scene graph nodes is a lot better.. found the x/y/z choices to be confusing and removing the dropdown down the line makes sense :+1: